### PR TITLE
Fix linting issues

### PR DESCRIPTION
**Pull request by TheiLLeniumStudios**
**Describe Pull request**
First, make sure you've read and are following the contribution guidelines and style guide and your code reflects that.
Write up a clear and concise description of what your pull request adds or fixes and if it's an added feature explain why you think it should be included in the core.

If your PR is to fix an issue mention that issue here

**Questions (please complete the following information):**
- Have you personally loaded this code into an updated qbcore project and checked all it's functionality? [yes/no] (Be honest)
- Does your code fit the style guidelines? [yes/no]
- Does your PR fit the contribution guidelines? [yes/no]

### CODE REVIEW

File: client/main.lua [hunk 1]
```diff
@@ -140,11 +140,9 @@ local function DestroyExitTarget()
 end
 
 local function GetDutyTargetText()
-  local text = nil
+  local text = '[E] Clock In'
```
Comment (BerkieBb) [inline client/main.lua:143]:
```suggestion
  local text = onDuty and '[E] Clock Out' or '[E] Clock In'
```

Comment (BerkieBb) [reply client/main.lua:143]: This is a better way to handle it instead of having an if statement under it

File: client/main.lua [hunk 2]
```diff
@@ -140,7 +140,7 @@ local function DestroyExitTarget()
 end
 
 local function GetDutyTargetText()
-  local text = '[E] Clock In'
+  local text = onDuty and '[E] Clock Out' or '[E] Clock In'
   if onDuty then
     text = '[E] Clock Out'
```
Comment (BerkieBb) [inline client/main.lua:145]: Remove this if statement

**Review by BerkieBb — CHANGES_REQUESTED** [review]

File: client/main.lua (hunk 2 above)
Comment (MatthewRorke) [reply client/main.lua]: I went full REEEE

**Review by BerkieBb — APPROVED** [review]

**Pull request merged**